odcockpit/src/plugins/environmentviewer/EnvironmentViewerGLWidget.cpp

```diff
@@ -71,7 +71,7 @@
 #include "opendavinci/odcore/wrapper/Eigen.h"
 #include "opendavinci/odcore/wrapper/SharedMemory.h"
 #include "opendavinci/odcore/wrapper/SharedMemoryFactory.h"
-#include "opendavinci/generated/odcore/data/SharedPointCloud.h"
+//#include "opendavinci/generated/odcore/data/SharedPointCloud.h"
```

Comment:
Because EnvironmentViewerGLWidget.h has already included this header. I removed this commented header here.